### PR TITLE
docs: Edits to remove jargon

### PR DESCRIPTION
**Pull request by TeriGlover**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: aio/content/guide/using-libraries.md [hunk 1]
```diff
@@ -8,10 +8,10 @@ See the [Angular Resources](resources) page for links to the most popular ones.
...
 
-The `ng add` command uses a package manager such as [npm](https://www.npmjs.com/) or [yarn](https://yarnpkg.com/) to install the library package, and invokes schematics that are included in the package to other scaffolding within the project code, such as adding import statements, fonts, themes, and so on.
+The `ng add` command uses a package manager such as [npm](https://www.npmjs.com/) or [yarn](https://yarnpkg.com/) to install the library package, and invokes schematics that are included in the package to other scaffolding within the project code, such as adding import statements, fonts, and themes.
 
-A published library typically provides a README or other documentation on how to add that lib to your app.
-For an example, see [Angular Material](https://material.angular.io/) docs.
+A published library typically provides a README or other documentation on how to add that lib to your application.
+For an example, see the [Angular Material](https://material.angular.io/) documents.
```
Comment (atscott) [inline aio/content/guide/using-libraries.md:14]: Isn't "docs" short for "documentation" in this context? Also, I think this link should probably be https://material.angular.io/guide/getting-started but maybe that's outside the scope of this PR.

**Review by atscott — APPROVED** [review]
reviewed-for: fw-router

**Review by AndrewKushnir — APPROVED** [review]
Looks good to me, thanks.

One quick comment: it'd be great to mention specific guides that were updated in commit message body and/or commit title.

File: aio/content/guide/using-libraries.md (hunk 1 above)
Comment (aikidave) [reply aio/content/guide/using-libraries.md:14]: I think `documentation` is better than `documents` here. @TeriGlover, any thoughts on this?

**Review by aikithoughts — APPROVED** [review]
One minor item. Otherwise, LGTM!

Reviewed-for: global-docs-approvers

**Review by IgorMinar — APPROVED** [review]
LGTM!

**Pull request closed without merging**